hr/specs/swagger.yaml

```diff
@@ -172,15 +174,16 @@ definitions:
         type: string
         format: date_time
       salary:
-        type: integer
```

Comment:
This, on the other hand... don't do this. 

If you want to include a $ dollar sign or commas, we can do that on the GET and convert this to a string... BUT for any incoming data (aka. technically only patch), turning salary into a string removes all of the critical validation that comes with an integer. While we may be the only ones to touch it for now, but it leaves open the opportunity to change an employee's salary to "feefifofum".